downtime/test_data/doc_configdb.json

```diff
@@ -0,0 +1,121 @@
+{
+    "count": 9,
+    "next": null,
+    "previous": null,
+    "results": [
+      {
+        "code": "tst",
+        "elevation": 100,
+        "enclosure_set": [
+          {
+            "code": "doma",
+            "telescope_set": [
+              {
+                "code": "1m0a",
+                "name": "1 Meter",
+                "instrument_set": [
+                  {
+                    "instrument_type": {
+                      "code": "1M0-SCICAM-SINISTRO"
```

Comment:
I'm not as against having these "lco" values because they don't have LCO in the name, so in my opinion they serve as just as good an example as "1M0-FAKE-INSTRUMENT-TYPE" would...

Reply:
My thought process is similar to Jon's. But for users who are familiar with LCO's nomenclature it may somehow indicate that the OCS is somehow intrinsically linked to LCO's naming conventions, which it's not. Was that your thought @eheinrich ?

I am happy to come up with some fake names/locations.

Any opposition to Sci-Fi references? ARK site for [Arrakeen](https://dune.fandom.com/wiki/Arrakeen), with a `2M0-SCICAM-CHOAM`. [CHOAM is the megacorp in the Dune universe](https://dune.fandom.com/wiki/CHOAM/XD)

I can also just use 1M0-FAKE-INSTRUMENT-TYPE...

Reply:
It's true that these two different strings might be equally meaningless to non-LCO people looking at the documentation. My thought process was more that if I were someone who was not familiar with LCO, I would wonder what a "sinistro" was and whether it was significant for some reason, and might go trying to find the answer, as opposed to just immediately accepting something like "1M0-INSTRUMENT-A" or the proposed "1M0-FAKE-INSTRUMENT-TYPE" as some example instrument type. Something that is more obviously just a made up name feels a little cleaner to me for documentation purposes. I don't really know much about sci-fi so don't have much of an opinion on that honestly, but again if I were looking at the docs I might wonder what a "choam" is. Maybe there are members of our potential user base that might appreciate the reference though?

Reply:
Although I will include that I once poorly chose to name something internal after Mark Watney, the hero of The Martian, which ended up rubbing some people the wrong way, so it might be safer to not pull from sci-fi.

Reply:
Haha, cool. Just wanted to throw that out there. We did some of that in the TOM Toolkit a while ago.

I'll make them obviously fake instruments.